genie-common/src/test/java/com/netflix/genie/common/dto/ApplicationStatusUnitTests.java

```diff
@@ -64,6 +64,6 @@ public void testInvalidApplicationStatus() throws GeniePreconditionException {
      */
     @Test(expected = GeniePreconditionException.class)
     public void testBlankApplicationStatus() throws GeniePreconditionException {
-        ApplicationStatus.parse(null);
```

Comment:
Isn't it worth testing both?

Reply:
The IDE/compiler checks the `@Nonnull` for us after `@ParametersAreNonnullByDefault` was added. it is now an opt out via `@Nullable`. Leaving the check in with `null` just produced warnings.